### PR TITLE
Fix spacing issue in resume and CI

### LINKED ISSUE #66: Odd Spacing When not Using Bullet Points for Resume Entries
If you write in paragraph form for a resume entry content (instead of bullet points), you get some weird spacing issues with the top and bottom margin. This should be made more consistent. 

image

Originally reported in the discord.

### PR DESCRIPTION
**Pull request by DeveloperPaul123**
## Fixed

- Spacing issue in resume when bulleted lists are not used (Resolves #66)
- Issue in CI where source-sans font family was not installed resulting in incorrect references being used for the tests.

### CODE REVIEW

File: lib.typ [hunk 1]
```diff
@@ -263,6 +263,10 @@
   ]
 
   show heading.where(level: 2): it => {
+    set block(
```
Comment (detuur) [inline lib.typ:266]: `set block` doesn't seem to do anything? I've messed around with the `above` and `below` values and they've produced identical results. Setting `fill: blue`, which should colour the entire block, also does nothing. I'm not fluent in typst yet but it seems to me it should be used like on line 54.

Comment (detuur) [reply lib.typ:266]: This also applies to the other uses of `set block` in this commit, as far as I can tell.

Comment (DeveloperPaul123) [reply lib.typ:266]: Hmm very interesting, it seems like you're right. If I remove all the `block` calls, things still look the same...

**Comment by DeveloperPaul123** [comment]
@detuur Ok I think I resolved the issues with the use of `block` and the spacing is looking a bit better now. I added some extra space to be between resume entry content and the headers. Let me know what you think.

**Pull request merged**